### ISSUE TITLE
Does not work with dailymotion

### ISSUE THREAD

**Issue by alfredalfie123** [berstend/puppeteer-extra#150]
After using latest version of puppeteer-extra-plugin-stealth, I found out that Dailymotion videos which have ads always show blank screen. Without that, it worked fine. 
Here is the sample Dailymotion URL which cause the issue: https://www.dailymotion.com/video/x7qyxew

**Comment by brunogaspar** [comment]
Do you have a code snippet of what you're doing?

**Comment by alfredalfie123** [comment]
@brunogaspar 
`
const network = require('network');
const puppeteer = require('puppeteer-extra');
const pluginStealth = require('puppeteer-extra-plugin-stealth');

const stealth = pluginStealth();

puppeteer.use(stealth);

async function launch_browser(launch_opt = {}) {
  try {
    const default_launch_opt = {
      headless: false,
      ignoreDefaultArgs: ['--enable-automation'],
      executablePath: '/usr/bin/google-chrome',
      args: [
        '--no-sandbox',
        '--disable-setuid-sandbox',
        // '--app=https://www.google.com/'
        // '--lang=bn-BD,bn'
      ],
    };
    launch_opt = Object.assign(default_launch_opt, launch_opt);

    const browser = await puppeteer.launch(launch_opt);
    const context = await browser.defaultBrowserContext();
    const page = await context.newPage();

    await page.goto('https://www.dailymotion.com/video/x7qyxew', {
      waituntil: 'networkidle0'
    });
  } catch (err) {
    console.log(err);
  }
}

(async () => {
  await launch_browser();
})();
`
After the ads load, it show blank page.

**Comment by ardalann** [comment]
I was having the same issue on WashingtonPost. Turns out it's caused by the `iframe.contentWindow` evasion in the Stealth plugin. Try disabling that.

**Comment by berstend** [comment]
Closing in favor of #137

TL;DR: The `iframe.contentWindow` evasion may brake some sites, disable it as a workaround.
